steal.lic

```diff
@@ -114,13 +117,11 @@ class Steal
   def bin_items
     return if @bin_items.empty?
 
-    walk_to 6017
-    move('tap knocker')
-    move('north')
+    walk_to @bin_location
 
     stow_hands
     @bin_items.each do |item|
-      if @slow_bin_speed
+      if @slow_bin_speed #currently only setup for crossing
```

Comment:
Yes, you could change the line to something like `bput("put #{item} in bin", 'nods toward you as your .* falls into the .* bin')`